CONTRIBUTING.md

```diff
@@ -89,6 +89,42 @@ After build completes, you can preview the resulting site locally using built-in
 	
 This will start the local preview on localhost:8080. You can then view the changes by going to http://localhost:8080/docs/[path] (e.g. http://localhost:8080/docs/getting-started/).   
 
-## Contributing to samples
+# Contributing to samples
 
-See the [Samples Readme](https://github.com/dotnet/core-docs/blob/master/samples/README.md).
+We encourage samples for any of our topics. Readers can download and run the samples. They
+can explore the code and experiment with their own changes.
+
+All samples should be complete applications or libraries. Where the sample creates a library,
+it should include unit tests or an application that lets readers run the code.
+
+The samples are organized in directories that match the directories under
```

Comment:
It's outside the context of your PR, but a great opportunity to raise the feedback and fix it.

I'd love to see us link to the Core contributing docs, since they cover some important and complementary content: https://github.com/dotnet/coreclr/blob/master/Documentation/project-docs/contributing.md.

> Before adding content, please submit an issue describing your content proposal.

We can just remove "before adding content,"

> Each author should have a link to an online presence for himself/herself.

"to their online presence" ... simplify the gender issue

> Step 1: Open an Issue describing the article you wish to write and how it relates to existing content. Get approval to write your article.

I'd prefer "Get feedback on your proposal." Step 1 (IMO) is less about getting a required ticket to proceed and more about the maintainers not feeling bad if someone skips step 1, invests a bunch of time in an article we opt not to merge. That is the "open source way", IMO.

> Don't forget to submit

3 of the points in the guidance checklist start out as "don't do blah". I suggest we tell folks what we want them to do. "Do blah".

I see: 

> If updating code samples in /samples/, be sure any line number references in your article remain correct

How about:

>  Be sure any line number references in your article remain correct if updating code samples in /samples/.

I see: 

> The documentation is built using docfx and Markdown. It is hosted on the .NET Core website.

How about:

> The documentation is written in [GitHub-flavored Markdown](https://help.github.com/categories/writing-on-github/) and built using [docfx](http://dotnet.github.io/docfx/). The documented is hosted at [docs.microsoft.com](https://docs.microsoft.com/dotnet).

we should include a statement on when docfx will move off of DNX to .NET Core 1.0

> The conceptual documentation is placed in the docs folder in the root of the repo so most of the work will happen there.

What "work"?

> Using the DNX-based docfx tool, you build the resulting

"the resulting" ???

> After build completes

After "the" build ...

> http://localhost:8080/docs/path

Does "path" take you to a specific location or a 404?